loops-nico-kat-tiff.js

```diff
@@ -0,0 +1,25 @@
+// Logging values with for loops
+// Write a for loop that logs each number from 1 - 20.
+// Write a for loop that logs the result of each number from 1 - 20 tripled.
+// Create a for loop that logs each even number from 1-20, and in the place of every odd number, returns the word "ODD" Expected output --> ODD, 2, ODD, 4, ODD, 6 ...etc
+for (var i = 0; i < array.length; i++) {
```

Comment:
Looks like you were setting up for loop. Considering this question, it has not requested an array nor ask to create an array. 
`a for loop that logs each even number from 1-20` so refactoring Line 5 for this question
`for(let i = 1; i <= 20; i++) {`

then consider conditional statements with the modulo operator to determine what to do for even values.